yb-voyager/cmd/importData_test.go

```diff
@@ -846,6 +853,636 @@ CREATE TABLE public.foo (
+
+	for {
+		if snapshotPhaseCompleted(t, postgresContainer.GetConfig().Password,
+			yugabytedbContainer.GetConfig().Password, 10, `test_schema."test_live"`) {
+			break
+		}
+		time.Sleep(1 * time.Second)
+	}
+
+	// Connect to both Postgres and YugabyteDB.
+	pgConn, err := postgresContainer.GetConnection()
+	testutils.FatalIfError(t, err, "connecting to Postgres")
+
+	ybConn, err := yugabytedbContainer.GetConnection()
+	testutils.FatalIfError(t, err, "Error connecting to YugabyteDB")
+
+	// Compare the full table data between Postgres and YugabyteDB for snapshot part.
+	// We assume the table "test_data" has a primary key "id" so we order by it.
+	if err := testutils.CompareTableData(ctx, pgConn, ybConn, "test_schema.test_live", "id"); err != nil {
+		t.Errorf("Table data mismatch between Postgres and YugabyteDB: %v", err)
+	}
+
+	//streaming events 5 events
+	postgresContainer.ExecuteSqls([]string{
+		`INSERT INTO test_schema.test_live (name, email, description)
+SELECT
+	md5(random()::text),                                      -- name
+	md5(random()::text) || '@example.com',                    -- email
+	repeat(md5(random()::text), 10)                           -- description (~320 chars)
+FROM generate_series(1, 5);`,
+	}...)
+
+	for {
+		if streamingPhaseCompleted(t, postgresContainer.GetConfig().Password,
+			yugabytedbContainer.GetConfig().Password, 5, `test_schema."test_live"`) {
+			break
+		}
+		time.Sleep(1 * time.Second)
+	}
+
+	// Compare the full table data between Postgres and YugabyteDB for streaming part.
+	if err := testutils.CompareTableData(ctx, pgConn, ybConn, "test_schema.test_live", "id"); err != nil {
+		t.Errorf("Table data mismatch between Postgres and YugabyteDB after streaming: %v", err)
+	}
+
+	// Run cutover
+	_, err = testutils.RunVoyagerCommand(nil, "initiate cutover to target", []string{
+		"--export-dir", exportDir,
+		"--yes",
+		"--prepare-for-fall-back", "false",
+	}, nil, false)
+	testutils.FatalIfError(t, err, "Cutover command failed")
+
+	metaDB, err = metadb.NewMetaDB(exportDir)
+	testutils.FatalIfError(t, err, "Failed to initialize meta db")
+
+	for {
+		status := getCutoverStatus()
+		if status == COMPLETED {
+			break
+		}
+		time.Sleep(1 * time.Second)
+	}
+
+	yugabytedbContainer.ExecuteSqls([]string{
+		`INSERT INTO test_schema.test_live (name, email, description)
+SELECT
+	md5(random()::text),                                      -- name
+	md5(random()::text) || '@example.com',                    -- email
+	repeat(md5(random()::text), 10)                           -- description (~320 chars)
+FROM generate_series(1, 10);`,
+	}...)
+
+	//Check if ids from 16-25 are present in target this is to verify the sequence serial col is restored properly till last value
+	validateSequence(t, 16, 25, ybConn, `test_schema.test_live`)
+
+}
+
+// test for live migration with resumption and failure during restore sequences
+func TestLiveMigrationWithImportResumptionOnFailureAtRestoreSequences(t *testing.T) {
```

Comment:
nit one more comment for each test - how/what it is exactly doing will help considering these tests are going to be a long text.